<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>327</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_11">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizeIncrement">
          <size>
           <width>9</width>
           <height>6</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Mongolian Baiti</family>
           <pointsize>20</pointsize>
           <bold>true</bold>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="text">
          <string>图表制作器</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="minimumSize">
        <size>
         <width>331</width>
         <height>191</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Mongolian Baiti</family>
        </font>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab1">
        <attribute name="title">
         <string>面积图</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <layout class="QHBoxLayout" name="horizontalLayout_5">
                   <item>
                    <widget class="QLabel" name="label_2">
                     <property name="sizePolicy">
                      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                       <horstretch>0</horstretch>
                       <verstretch>0</verstretch>
                      </sizepolicy>
                     </property>
                     <property name="font">
                      <font>
                       <family>Mongolian Baiti</family>
                       <pointsize>10</pointsize>
                      </font>
                     </property>
                     <property name="layoutDirection">
                      <enum>Qt::RightToLeft</enum>
                     </property>
                     <property name="text">
                      <string>数值设置</string>
                     </property>
                     <property name="alignment">
                      <set>Qt::AlignCenter</set>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer_2">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                   <item>
                    <spacer name="horizontalSpacer">
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                     <property name="sizeHint" stdset="0">
                      <size>
                       <width>40</width>
                       <height>20</height>
                      </size>
                     </property>
                    </spacer>
                   </item>
                  </layout>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout">
                   <item>
                    <widget class="QSplitter" name="splitter_2">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <widget class="QWidget" name="horizontalLayoutWidget_2">
                      <layout class="QHBoxLayout" name="horizontalLayout_2">
                       <item>
                        <widget class="QLabel" name="label_8">
                         <property name="text">
                          <string>Y轴范围：</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_9">
                         <property name="text">
                          <string>最小值</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QDoubleSpinBox" name="doubleSpinBox_3"/>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_10">
                         <property name="text">
                          <string>最大值</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QDoubleSpinBox" name="doubleSpinBox_4"/>
                       </item>
                       <item>
                        <widget class="QPushButton" name="pushButton_3">
                         <property name="font">
                          <font>
                           <family>Mongolian Baiti</family>
                           <bold>true</bold>
                          </font>
                         </property>
                         <property name="text">
                          <string>设置</string>
                         </property>
                         <property name="default">
                          <bool>true</bool>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                     <widget class="QWidget" name="horizontalLayoutWidget">
                      <layout class="QHBoxLayout" name="horizontalLayout">
                       <item>
                        <widget class="QLabel" name="label_5">
                         <property name="text">
                          <string>X轴范围：</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_6">
                         <property name="text">
                          <string>最小值</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_7">
                         <property name="text">
                          <string>最大值</string>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QDoubleSpinBox" name="doubleSpinBox_2"/>
                       </item>
                       <item>
                        <widget class="QPushButton" name="pushButton_2">
                         <property name="font">
                          <font>
                           <family>Mongolian Baiti</family>
                           <bold>true</bold>
                          </font>
                         </property>
                         <property name="text">
                          <string>设置</string>
                         </property>
                         <property name="default">
                          <bool>true</bool>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </widget>
                   </item>
                   <item>
                    <widget class="QSplitter" name="splitter">
                     <property name="orientation">
                      <enum>Qt::Vertical</enum>
                     </property>
                     <widget class="QWidget" name="horizontalLayoutWidget_3">
                      <layout class="QHBoxLayout" name="horizontalLayout_3">
                       <item>
                        <widget class="QLabel" name="label_11">
                         <property name="text">
                          <string>序列1  ：</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignCenter</set>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_12">
                         <property name="text">
                          <string>X 值</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignCenter</set>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QDoubleSpinBox" name="doubleSpinBox_5"/>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_13">
                         <property name="text">
                          <string>Y 值</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignCenter</set>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QDoubleSpinBox" name="doubleSpinBox_6"/>
                       </item>
                       <item>
                        <widget class="QPushButton" name="pushButton_4">
                         <property name="font">
                          <font>
                           <family>Mongolian Baiti</family>
                           <bold>true</bold>
                          </font>
                         </property>
                         <property name="text">
                          <string>添加</string>
                         </property>
                         <property name="default">
                          <bool>true</bool>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                     <widget class="QWidget" name="horizontalLayoutWidget_4">
                      <layout class="QHBoxLayout" name="horizontalLayout_4">
                       <item>
                        <widget class="QLabel" name="label_14">
                         <property name="text">
                          <string>序列2 ：</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignCenter</set>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_15">
                         <property name="text">
                          <string>X 值</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignCenter</set>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QDoubleSpinBox" name="doubleSpinBox_7"/>
                       </item>
                       <item>
                        <widget class="QLabel" name="label_16">
                         <property name="text">
                          <string>Y 值</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignCenter</set>
                         </property>
                        </widget>
                       </item>
                       <item>
                        <widget class="QDoubleSpinBox" name="doubleSpinBox_8"/>
                       </item>
                       <item>
                        <widget class="QPushButton" name="pushButton_5">
                         <property name="font">
                          <font>
                           <family>Mongolian Baiti</family>
                           <bold>true</bold>
                          </font>
                         </property>
                         <property name="text">
                          <string>添加</string>
                         </property>
                         <property name="default">
                          <bool>true</bool>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </widget>
                    </widget>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_6"/>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QGridLayout" name="gridLayout_2">
               <item row="0" column="1">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="1">
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="1" column="0">
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item row="0" column="0">
                <widget class="QLabel" name="label_17">
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout_8">
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="font">
                    <font>
                     <family>Mongolian Baiti</family>
                     <pointsize>10</pointsize>
                    </font>
                   </property>
                   <property name="text">
                    <string>图表展示</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_5">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <spacer name="horizontalSpacer_6">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton">
                 <property name="font">
                  <font>
                   <family>Leelawadee UI</family>
                   <pointsize>10</pointsize>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::ClickFocus</enum>
                 </property>
                 <property name="contextMenuPolicy">
                  <enum>Qt::ActionsContextMenu</enum>
                 </property>
                 <property name="text">
                  <string>清除</string>
                 </property>
                 <property name="icon">
                  <iconset theme="audio-volume-medium">
                   <normaloff>.</normaloff>.</iconset>
                 </property>
                 <property name="autoDefault">
                  <bool>false</bool>
                 </property>
                 <property name="default">
                  <bool>true</bool>
                 </property>
                 <property name="flat">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QChartView" name="graphicsView"/>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>条形图</string>
        </attribute>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
